scripts/data-downloader

```diff
@@ -0,0 +1,5 @@
+#!/usr/bin/env python
```

Comment:
Could the scripts be renamed to begin with `wm-`? I have followed this pattern to avoid colliding names with other libraries that might have similar utilities.